client/js/lounge.js

```diff
@@ -104,7 +131,11 @@ $(function() {
 		return false;
 	}
 
-	viewport.on("contextmenu", ".user, .network .chan", function(e) {
+	viewport.on("contextmenu", ".network .chan", function(e) {
+		return showContextMenu(this, e);
+	});
+
+	viewport.on("click contextmenu", ".user", function(e) {
```

Comment:
I was very worried about the dropdown menu on left click, but it actually works very nicely. Nice job!
Especially like the fact that it's super snappy. On Slack it's not only slightly delayed but there is also an animation. Frustrating experience.